**What was tried:** QA workflow run [#25593509917](https://github.com/svv2014/loop/actions/runs/25593509917) on PR #241 (branch `fix/issue-236-loop`).

**What failed:**
```
No structured test names found
```

<details><summary>Last 20 lines of failure output</summary>

```
qa-merge	Run tests	2026-05-09T05:59:16.9312968Z ok 334 validate: valid minimal workflow passes
qa-merge	Run tests	2026-05-09T05:59:17.0754413Z ok 335 validate: all committed starter workflows pass validation
qa-merge	Run tests	2026-05-09T05:59:17.1380972Z ok 336 validate: missing version field fails
qa-merge	Run tests	2026-05-09T05:59:17.2009105Z ok 337 validate: version=2 fails
qa-merge	Run tests	2026-05-09T05:59:17.2642094Z ok 338 validate: name mismatch with filename fails
qa-merge	Run tests	2026-05-09T05:59:17.3276136Z ok 339 validate: duplicate trigger_label within issue_stages fails
qa-merge	Run tests	2026-05-09T05:59:17.3907732Z ok 340 validate: same trigger_label across issue_stages and pr_stages is allowed
qa-merge	Run tests	2026-05-09T05:59:17.4542529Z ok 341 validate: unknown transition target fails
qa-merge	Run tests	2026-05-09T05:59:17.5175392Z ok 342 validate: transition to terminal label 'done' passes
qa-merge	Run tests	2026-05-09T05:59:17.5809767Z ok 343 validate: decisions approve/reject referencing unknown labels fails
qa-merge	Run tests	2026-05-09T05:59:17.6442697Z ok 344 validate: empty workflow with no stages fails
qa-merge	Run tests	2026-05-09T05:59:17.7085206Z ok 345 validate: stage missing trigger_label fails
qa-merge	Run tests	2026-05-09T05:59:17.7717475Z ok 346 validate: stage missing handler fails
qa-merge	Run tests	2026-05-09T05:59:17.8356268Z ok 347 validate: missing handler script produces warning but passes
qa-merge	Run tests	2026-05-09T05:59:18.0080764Z ok 348 default.yaml: every referenced label is canonical
qa-merge	Run tests	2026-05-09T05:59:18.0829730Z ok 349 minimal.yaml: every referenced label is canonical
qa-merge	Run tests	2026-05-09T05:59:18.1774457Z ok 350 docs-only.yaml: every referenced label is canonical
qa-merge	Run tests	2026-05-09T05:59:18.3315977Z ok 351 all shipped workflows pass loop_workflow_validate
qa-merge	Run tests	2026-05-09T05:59:18.5762433Z ok 352 no deprecated alias appears as a workflow label
qa-merge	Run tests	2026-05-09T05:59:18.6086615Z ##[error]Process completed with exit code 1.
```
</details>

**Options:**
(a) Fix the failing test(s) — update the code or test to make them pass, then push a new commit to this PR.
(b) Skip the test(s) — if the test is flaky or irrelevant to this change, annotate it with the appropriate skip marker and document why.

--- a/lib/env.sh
+++ b/lib/env.sh
@@ -50,11 +50,3 @@ loop_run_orchestrator() {
             "$prompt"
     fi
 }
-
-# Helper: send notification (no-op if not configured)
-loop_notify() {
-    local msg="$1"
-    if [ -n "$LOOP_NOTIFY" ]; then
-        eval $LOOP_NOTIFY "\"$msg\"" 2>/dev/null || true
-    fi
-}

